### DISCUSSION TITLE
Featured Image, Custom Folder and Author Avatar.

### DISCUSSION THREAD

**Discussion by iEscapedVim** [decaporg/decap-cms#6919] [Q&A]
Hi 

So I'm trying to explore DecapCMS but have a few things I couldn't find.

1. First of all my posts/news structure is like this

```
content/
├── news(posts folder)/
│   └── posts-title/
│             └── img/
│             │   └── cover.png
│             │   └── avatar.png
│             └── index.md
```

but Decap creates posts like this 

```
content/
├── news(posts folder)/
│     └── index.md
```

How can I configure it, so Decap creates folders like I want?

3. I need to know how I can set cover/featured images for my posts.
4. I want to be able to select an image for the author and also his/her name.
5. Also the ability to add categories and tags. just like in Hugo, where I can just write a category. 

![230922_11h28m09s_screenshot](https://github.com/decaporg/decap-cms/assets/32537564/f6557d9e-2d88-4c87-8a1d-5701c3cc05e2)
![230922_11h29m33s_screenshot](https://github.com/decaporg/decap-cms/assets/32537564/5d2cb643-0b4e-48a0-834f-1f8d1bc74242)

![1](https://github.com/decaporg/decap-cms/assets/32537564/66a57be0-f45b-46b6-bb82-2493c8662326)

**Comment by martinjagodic** [comment]
1. Add this to your collection config:

`path: '{{slug}}/index'

more here: https://decapcms.org/docs/beta-features/#folder-collections-path

Regarding your other points: your question is too vague. Please write what you tried yourself and your config where it didn't work.
